pombola/hansard/api/views.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import Prefetch
+from django.shortcuts import get_object_or_404
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from ..models import Entry, Sitting, Source, Venue
+from .serializers import (
+    EntrySerializer, SittingSerializer, SittingWithEntriesSerializer,
+    SourceSerializer, VenueSerializer
+)
+
+
+class EntryViewSet(viewsets.ModelViewSet):
+    queryset = Entry.objects.order_by('id')
+    serializer_class = EntrySerializer
+
+
+class SittingViewSet(viewsets.GenericViewSet):
+
+    queryset = Entry.objects.order_by('id')
```

Comment:
If you don't have it, then going to /api/v0.1/hansard/sittings/ gives you this error:

```
'SittingViewSet' should either include a `queryset` attribute, or override the `get_queryset()` method.
```

The traceback is:

```
Traceback:
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  137.                 response = response.render()
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/django/template/response.py" in render
  103.             self.content = self.rendered_content
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/rest_framework/response.py" in rendered_content
  71.         ret = renderer.render(self.data, media_type, context)
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/rest_framework/renderers.py" in render
  675.         context = self.get_context(data, accepted_media_type, renderer_context)
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/rest_framework/renderers.py" in get_context
  654.             'filter_form': self.get_filter_form(data, view, request),
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/rest_framework/renderers.py" in get_filter_form
  592.         queryset = view.get_queryset()
File "/home/mark/.virtualenvs/pombola-1.7/local/lib/python2.7/site-packages/rest_framework/generics.py" in get_queryset
  67.             % self.__class__.__name__

Exception Type: AssertionError at /api/v0.1/hansard/sittings/
Exception Value: 'SittingViewSet' should either include a `queryset` attribute, or override the `get_queryset()` method.
```